spec/features/invoices/pdfs/new_spec.rb

```diff
@@ -0,0 +1,101 @@
+require 'rails_helper'
+
+feature 'New Invoice PDF' do
+  let(:employee) { create :employee }
+  let(:customer) { create :customer, confidential_title: 'Lölimann' }
+  let(:invoice_attributes) do
+    {
+      customer: customer,
+      employee: employee,
+      vat_rate: 0.08,
+      date: '2017-10-01'
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.